HA-Lambda-WP_configuration.yaml

```
@@ -27,7 +24,7 @@ modbus:
           address: 0002
           input_type: holding
           unit_of_measurement: "°C"
-          state_class: total
+          state_class: measurement
```

```suggestion
          state_class: measurement
          device_class: temperature
```

same for other temp values around here

---

I applied your changes, but not the typos ;-)